docs/index.md

```diff
@@ -1,54 +1,75 @@
-<pre>aws ec2 describe-instances --profile foo --query 'Reservations[].Instances[].InstanceId' --filters Name=tag-key,Values=owner --output text</pre>
-<pre>docker ps --filter “label=owner”</pre>
-</details>
-
-If not, then try clicking on the arrows to see the recommended way of doing those tasks. Does it bother you that each of those is a bespoke, cryptic incantation of various vendor-specific tools? It's a lot of commands you have to use, applications you need to install, and DSLs you have to learn just to do some pretty basic tasks. In Wash, these basic tasks are simple. You'll find that
-
-<details>
-<summary>Listing stuff is as easy as <code>ls</code></summary>
-<pre>ls aws/foo/resources/ec2/instances</pre>
-<pre>ls kubernetes/foo/bar/pods</pre>
-</details>
-<details>
-<summary>Reading stuff is as easy as <code>cat</code>'ing a file</summary>
-<pre>cat gcp/foo/compute/bar/console.out</pre>
-<pre>cat aws/foo/resources/s3/bar/baz</pre>
-</details>
-<details>
-<summary>Execing a command is as easy as <code>wexec</code></summary>
-<pre>wexec kubernetes/foo/bar/pods/baz uname</pre>
-<pre>wexec gcp/foo/compute/bar uname</pre>
-</details>
-<details>
-<summary>Finding stuff is as easy as <code>find</code></summary>
-<pre>find aws/foo -k '*ec2*instance' -meta '.tags[?].key' owner</pre>
-<pre>find docker -k '*container' -meta '.labels.owner' -exists</pre>
-</details>
-
-And this is only scratching the surface of Wash's capabilities. Check out the screencast below
-
-<script id="asciicast-MkbZKZZcmokHQ8z8jTv0PQs9F" src="https://asciinema.org/a/mX8Mwa75rr1bJePLi3OnIOkJK.js" async></script>
-
-and the [tutorials](tutorials) to learn more.
+
+<p class="intro-normal">A UNIX-like shell for managing your cloud native and non-cloud native things.</p>
+
+<a href="{{ '/getting_started' | relative_url }}" style="font-size: 22px;">
+<strong>Get started</strong>
+</a>
+
+<p class="intro-large"><strong>With Wash</strong></p>
+
+* <p class="intro-normal">You can use <code>ls</code> to list, <code>cat</code> to read, and <code>wexec</code> to run commands on all your things. No more switching between confusing CLI tools.</p>
+* <p class="intro-normal">You can use <code>find</code> to filter anything on anything. No more complicated query DSLs.</p>
+* <p class="intro-normal">You can <code>cd</code> through a vendor's API. No more navigating complex console UIs.</p>
+
+<p class="intro-large"><strong>See for yourself</strong></p>
+
+<!-- Display the demos -->
+<select id="demo">
+  <option value="aws">AWS</option>
+  <option value="gcp">GCP</option>
+  <option value="kubernetes">Kubernetes</option>
+  <option value="docker">Docker</option>
+  <option value="external-plugin">Other</option>
+</select>
+<div id="aws-demo" style="display: none;">
+  {% capture aws_annotation %}
+  The EC2 instance <code>find</code> query shown above (<code>find . -k '*instance' -m '.state.name' running -m '.tags[?].key' owner</code>) returns all running EC2 instances with the 'owner' tag.
```

Comment:
Re: https://github.com/puppetlabs/wash/issues/674, the long VM `find` queries are the only things that get cut off on a full-screen, mobile-device screencast view, but people can still see the full expression via the annotation. We could look into making the screencasts work on mobile (without them having to fullscreen) but I'd like to do that separately because what's in this PR is good enough.